src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -25,23 +26,38 @@ public class AnalysisService : IDisposable
     {
         #region Private Fields
 
+        /// <summary>
+        /// Maximum number of runspaces we allow to be in use for script analysis.
+        /// </summary>
         private const int NumRunspaces = 1;
 
+        /// <summary>
+        /// Name of the PSScriptAnalyzer module, to be used for PowerShell module interactions.
+        /// </summary>
         private const string PSSA_MODULE_NAME = "PSScriptAnalyzer";
 
+        /// <summary>
+        /// Provides logging.
+        /// </summary>
         private ILogger _logger;
-        private RunspacePool _analysisRunspacePool;
 
-        private bool _hasScriptAnalyzerModule;
+        /// <summary>
+        /// Runspace pool to generate runspaces for script analysis and handle
+        /// ansynchronous analysis requests.
+        /// </summary>
+        private RunspacePool _analysisRunspacePool;
 
-        private string[] activeRules;
-        private string settingsPath;
+        /// <summary>
+        /// Info object describing the PSScriptAnalyzer module that has been loaded in
+        /// to provide analysis services.
+        /// </summary>
+        private PSModuleInfo _pssaModuleInfo;
 
         /// <summary>
         /// Defines the list of Script Analyzer rules to include by default if
         /// no settings file is specified.
         /// </summary>
-        private static readonly string[] IncludedRules = new string[]
+        private static readonly string[] s_includedRules = new string[]
```

Comment:
What is the `s` in there (I guess a typo, not Hungarian notation).
Just a though in general as I see that you change the style in general to be more consistent. You should make sure that those changes make it into version 1.x of the extension so that when you branch off to version 2.0, it is still easily possible to merge back commits to version 1.x

Reply:
The `s_` is to indicate a private static variable - so I suppose it is a sort of Hungarian-notation-lite - in the same vein as the `_` before private variables. It's the convention used in all the other Microsoft code I've seen. There are places within this codebase where it's not done as much, but I think we should change it over as we touch the code for both clarity and consistency with related projects.

I don't quite follow the latter part of your comment, but if you're saying we should try and make as much as possible work with 1.x versions, I am strongly in agreement. I don't think the v2 branch will be taking in commits from master for a little while yet. I'm not sure it's worth one big commit to change the style over - I imagine opinions might differ on that - although I'd be happy to do it personally.

Reply:
As, I see, I have never seen it this way (or probably I've just never recognized it). To re-word my other comment, which was more about the workflow in the (distant) future: Style changes should not be made any more once v2 has actually branched off (i.e. has become the mainstream branch) to allow for merge-ability of commits in v2 back to v1 (or style changes should be applied to both branches otherwise).

Reply:
Yes agreed. My opinion there then is that we should invest in a sweeping style change PR. The style in PSES is already inconsistent but is moving over to be much closer to the conventions recommended by Microsoft and used in the other PowerShell projects (PascalCase properties, `_`-prefixed private variables, etc.). Otherwise we'll be stuck with code that is less familiar and (in my opinion) not as clear.